docs/install.installing.rst

```diff
@@ -1,5 +1,6 @@
 .. # Copyright (C) 2020-2021 Intel Corporation
 .. # SPDX-License-Identifier: Apache-2.0
+.. Redundant, so removed from TOC
```

Comment:
If this file is redundant, it should just be removed

Reply:
I've removed the file.